homepage/index.html

```diff
@@ -0,0 +1,214 @@
+            </div>
+            <nav class="navigation">
+                <ul>
+                    <li><a href="#AboutAccess">AboutAccess</a></li>
+                    <li><a href="#Courses">Courses</a></li>
+                    <li><a href="#i-campus"> <i>i</i>-campus</a></li>
+                    <li><a href="#Apply"><button>
+                                <p>Apply</p>
+                            </button></a></li>
+                </ul>
+            </nav>
+        </div>
+
+        <div class="heading">
+            <div class="icons">
+                <img src="/homepage/images/access_.png" alt="">
+                <!-- <p>access</p>
+                <span>_</span> -->
+            </div>
+            <div class="burger">
+                <div class="vector1"><img src="/homepage/images/Vector.png" alt=""></div>
+            </div>
+        </div>
+    </header>
+
+
+
+
+    <div class="banner-image">
+        <div class="choose"><span>Choose your Path Wisely</span>
+            <p>Make This Monsoon , A ladder to success. Weekend Courses on Web Development, Mobile App Development
+                and UX/UI
+                Design starting in June and July.</p>
+        </div>
+    </div>
+
+
+    <div class="why-access">
+        <div class="why-access-text">
+            <p>Why access?</p>
+        </div>
+        <div class="wrapper">
+            <div class="community">
+                <img src="/homepage/images/communitylearning.png" alt="">
+                <span>COMMUNITY LEARNING</span>
+                <p>At access, you can learn with like-minded students who are eager to learn in an immersive environment
+                    just like you. Become part of a community of extremely passionate alumni.</p>
+            </div>
+            <div class="career">
+                <img src="/homepage/images/book.png" alt="">
+                <span>CAREER-FOCUSED CURRICULUM</span>
+                <p>You can learn what you need to know to start a career in the tech industry with our career-focused
+                    curriculum, par with industry standards.</p>
+            </div>
+            <div class="real">
+                <img src="/homepage/images/globe.png" alt="">
+                <span>REAL-WORLD SKILLS</span>
+                <p>At access, you will learn to answer the problems by developing solutions for real - time problems in
+                    the
+                    world. Also, you shall have a chance to interact with industry experts as well.</p>
+            </div>
+        </div>
+
+    </div>
+
+    <div class="course-layout">
+        <div class="layout1">
+            <span>Request A Course Layout</span>
+            <p>Go through our Course Layout, and see what we have to offer you. In the Course Layout, we explore:</p>
+            <div class="bullets">
+                <ul>
+                    <li> Curriculum, programming languages, and technology covered</li>
```

Comment:
1. The orange color bullet points are not looking proper 
2. and align left has per above para

<img width="325" alt="Screenshot 2021-09-17 at 8 35 25 AM" src="https://user-images.githubusercontent.com/47788183/133717681-a2442ae1-9044-4d3a-9c99-1a9018d7b0c4.png">

Reply:
done